src/caret_analyze/value_objects/callback.py

```diff
@@ -64,7 +64,13 @@ def type_name(self) -> str:
 
 
 class CallbackValue(ValueObject, metaclass=ABCMeta):
-    """Callback value base class."""
+    """
+    Value object class for representing a callback.
+
+    This class has minimal information and no structure,
+    and used as the return value of ArchitectureReader.
```

Comment:
It's better to split it into two sentences because it holds two contents

Reply:
addressed in : 0729012